data/autoyast_sle15/autoyast_firstboot.xml

```diff
@@ -95,4 +118,32 @@
         <timeout config:type="integer">0</timeout>
       </yesno_messages>
     </report>
+    <scripts>
+        <chroot-scripts config:type="list">
+            <script>
+            <chrooted config:type="boolean">true</chrooted>
+                <source>
+                    <![CDATA[#!/bin/bash
```

Comment:
it would reduce complexity to have the file in data and append only the export for YUI_HTTP_PORT. wdyt?

Reply:
Could be but I don't think it simplifies much as we will still need a script section to download the file, add that export and "chmod" it, so in the end we would have just as many lines of code. What may work though, is to merge the two "script" sections, I did not really polish that part. But at least, it works.

Reply:
So After trying, no, we cannot even merge those two script sections, it needs to be the way it is. the first script runs before reboot, and the second after.